generate_dockerfiles.py

```diff
@@ -0,0 +1,147 @@
+import os
```

Comment:
1. Files of this length/complexity should have a source file header
2. Would be good to have some diagnostic logging in this file to help track down any misbehavior in action

Reply:
Added the source file header